resources/assets/js/videos/components/videoScreen.vue

```diff
@@ -452,10 +452,18 @@ export default {
         extractAnnotationFromFeature(feature) {
             return feature.get('annotation');
         },
-        handleFeatureSelect(e) {
+        handleFeatureSelect() {
+            // Don't use event.deselected here because it uses selections in current frame only,
+            // see https://github.com/biigle/core/issues/552
+
+            // New selected annotations
+            let selected = this.selectInteraction.getFeatures().getArray().map(this.extractAnnotationFromFeature);
+            // Old selected annotations
+            let deselected = this.annotations.filter((a) => {return a.selected;})
+                                             .filter((a) => {return !selected.includes(a)});
```

Comment:
You can use syntactic sugar:

```suggestion
            let deselected = this.annotations.filter(a => a.selected)
                                             .filter(a => !selected.includes(a));
```